### ISSUE TITLE
Question about PyramidBox loss

### ISSUE THREAD

**Issue by lijiannuist** [Goingqs/PyramidBox#1]
formula(5) in original paper shows that we need to define new target box coordinates of head. however, when I check your code, I can not find the definition.

**Comment by Goingqs** [comment]
Yes, that's for convenience. You can follow the original paper but I don't think that will help a lot.

**Comment by Edwardmark** [comment]
@lijiannuist And I cannot find where the code match head and gt, I cannot find the downsample operation to match face gt with head anchor, can you help me?Thanks.
